explainer.md

```diff
@@ -74,3 +93,27 @@ Creating an API that provides information about the real world has privacy impli
 *   The geometry of a known location may allow a site to identify the user's location
 
 In many ways these privacy implications are similar to video (camera) inputs and should be considered to have similar privacy implications. However, the implications are not necessarily equivalent. For example, users may consider their image to be more private than the layout of their room while a mesh of one's face may allow easier fingerprinting than RGB image processing.
+
+# API IDL
+```
+[SecureContext, Exposed=Window] interface XRHitResult {
+  readonly attribute Float32Array poseModelMatrix;
+};
+
+partial interface XRSession {
+  Promise<FrozenArray<XRHitResult>> requestHitTest(Float32Array origin, Float32Array direction, XRFrameOfReference frameOfReference);
```

Comment:
I think this is OK for now, but we'll want to make sure the ray input format here tracks whatever decision is made in https://github.com/immersive-web/webxr/issues/339

Reply:
Will do - i'll keep tabs on that issue and update as appropriate.